README.md

```diff
@@ -30,23 +30,76 @@
 | darwin-ppc | [![Build Status](https://dev.azure.com/aros-development-team/AROS/_apis/build/status/aros-development-team.AROS-darwin-ppc?branchName=master)](https://dev.azure.com/aros-development-team/AROS/_build/latest?definitionId=25&branchName=master) | --- |
 | mingw32-i386 | [![Build Status](https://dev.azure.com/aros-development-team/AROS/_apis/build/status/aros-development-team.AROS-mingw32-i386?branchName=master)](https://dev.azure.com/aros-development-team/AROS/_build/latest?definitionId=23&branchName=master) | --- |
 
+## Building from sources
+
+### Requirements
+
+See [aros.org](http://www.aros.org/documentation/developers/compiling.php#getting-the-needed-packages)
+
+**20200315 UPDATE**: comment every occurrence of `#MM` in `./workbench/devs/keymaps/mmakefile.src`
```

Comment:
If I dont comment that metamake file, the Docker build fails when that file is reached (unfortunately don't have a log at hand but can provide it).

Any idea why? Am I missing anything, maybe a dependency?